codebase/src/cpp/syscommon/include/syscommon/Exception.h

```diff
@@ -31,7 +31,7 @@ namespace syscommon
 
 			}
 
-			virtual ~Exception() throw ()
+			virtual ~Exception() noexcept
```

Comment:
Can this `noexcept` clause be removed outright? I'm not sure why I had an empty `throw()` declaration in the first place.

Reply:
Destructors should always be noexcept, having this specification here neither helps nor hinders, as a destructor throwing is almost always undefined behaviour anyway.
With this specifier here, the application just knows to terminate (which is probably what happens without the specifier anyway)